libsolidity/analysis/DocStringAnalyser.cpp

```diff
@@ -36,7 +36,7 @@ using namespace solidity::frontend;
 namespace
 {
 
-void copyMissingTags(StructurallyDocumentedAnnotation& _target, set<CallableDeclaration const*> const& _baseFunctions)
+void copyMissingTags(set<CallableDeclaration const*> const& _baseFunctions, StructurallyDocumentedAnnotation& _target, CallableDeclaration const* _callable = nullptr)
```

Comment:
Would it make sense to change the first argument from a set to just `CallableDeclaration const* _baseFunction`?

Reply:
From the perspective of the function yes. From the perspective of the caller no, because in 50% of calls I have that parameter only as `set` and would need to do the size check before the call. This way I do the size check only once in the function.

Reply:
True. So maybe not worth the change.